MantidQt/MantidWidgets/src/InstrumentView/ObjCompAssemblyActor.cpp

```diff
@@ -28,7 +28,7 @@ namespace MantidQt
 
           ObjCompAssembly_const_sptr objAss = getObjCompAssembly();
           mNumberOfDetectors = objAss->nelements();
```

Comment:
I thought that originally too but it turns out to be a protected member `ICompAssemblyActor.h`! A quick search reveals that it is referenced by all of the other actor classes and I'm unsure of the consequences of changing that. 

I think a proper code review of the instrument view code would be wise at some point.